almalinux-deploy.sh

```diff
@@ -360,6 +362,16 @@ grub_update() {
     fi
 }
 
+# Check do we have custom kernel (e.g. kernel-uek) and print warning
+check_custom_kernel() {
+output=$(rpm -qa | grep kernel-uek) || :
+if [ -n "${output}" ]; then
+    echo -e "[31;1mYou've had kernels that now can't be booting[0m:"
```

Comment:
I think it's not informative message for user. I suggest to add info about what should do a client in this case and why its server can't be booting